Fix paths to files.

Capstan opened the pull request:
When viewed at https://github.com/googleapis/aip, all of these links 404 otherwise.

Capstan committed:
Fix paths to files.

When viewed at https://github.com/googleapis/aip, all of these links 404 otherwise.

lukesneeringer reviewed (changes requested):
Thanks for doing this, @Capstan!

Changing this seems to make the links break on the rendered site.
Let me see if I can figure out the correct incantation.

Commit:
[fix] Use relative links to the Markdown file.

Links need to be relative links to the appropriate Markdown
_file_ in order to be able to be followed both in GitHub and
the rendered Jekyll site.

lukesneeringer reviewed (approved):
I figured out the right thing that works in both environments.

Thanks again for doing this!
